virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc

```diff
@@ -1,11 +1,11 @@
 [id="virt-importing-rhv-vm"]
-= Importing a single Red Hat Virtualization virtual machine
+= Importing a single {rh-virtualization} virtual machine
 include::modules/virt-document-attributes.adoc[]
 :context: virt-importing-rhv-vm
 :virt-importing-rhv-vm:
 toc::[]
 
-You can import a single Red Hat Virtualization (RHV) virtual machine into your {product-title} cluster by using the virtual machine wizard or the CLI.
+You can import a single {rh-virtualization-first} virtual machine into your {product-title} cluster by using the virtual machine wizard or the CLI.
```

Comment:
For some reason this one instance of the `{rh-virtualization-first}` common attribute is not rendering in my local `asciibinder build` preview. Instead, I am seeing the actual attribute rather than its value. I am not sure why, because it looks correct to me. All of the other instances of this common attribute in the PR render correctly as "Red Hat Virtualization (RHV)" in my local preview. Please test this in your preview.

Reply:
I rebuilt the doc but can't reproduce the issue you describe. There may be an issue with your environment. I'll check the final build to ensure this issue doesn't pop up again there.

Reply:
Thank you for checking this Rolfe!